apps/src/lab2/views/ExtraLinksModal.tsx

```diff
@@ -27,10 +31,25 @@ const ExtraLinksModal: React.FunctionComponent<ExtraLinksModalProps> = ({
   const [clonedLevelName, setClonedLevelName] = useState('');
   const [cloneError, setCloneError] = useState('');
   const [deleteError, setDeleteError] = useState('');
+  const [featuredProjectStatus, setFeaturedProjectStatus] = useState<
+    string | undefined
+  >('');
+
+  const channelId: string | undefined = useAppSelector(
+    state => state.lab.channel && state.lab.channel.id
+  );
+
+  const isStandaloneProject = levelLinkData.is_standalone_project;
```

Comment:
minor: I think we already have access to this via lab2Redux in `state.lab.levelProperties.isProjectLevel`?

(though @molly-moen I think you included this on the backend in `levels_controller#extra_links` - would `is_project_level`/`isProjectLevel` also work here, or is there a reason we need to do a separate lookup?)

Reply:
Oh good call, I think we could use `isProjectLevel` instead.